real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/ui/GpsPanel.java

```diff
@@ -119,11 +169,8 @@ private void addCacheToTree() {
 			
 			FilenameFilter filter = (dir, name) -> {
                 // TODO Auto-generated method stub
-                if(name.contains("png")) {
-                    return true;
-                }
-                return false;
-            };
+				return name.contains("png");
```

Comment:
this is for the filename filter to get only the map cache files out of that directory, it's an inner class